sagemaker-python-sdk/scikit_learn_randomforest/Sklearn_on_SageMaker_end2end.ipynb

```diff
@@ -0,0 +1,669 @@
+{
```

Comment:
<p>Can this not use the map that is maintained in the Python SDK? Otherwise this will have to be updated for every new region launch correct?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/793/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
The map is only being used for the boto3 only example, I believe.

Reply:
yes the map is only for the boto3 example. Goal is to show how to run sklearn training in plain boto3, eg something one would do in lambda